podcast_agent/client/transcriber.py

```diff
@@ -32,14 +33,49 @@ def read_from_url(self, url):
                     temp_file.write(audio_file.content)
                     temp_file_name = temp_file.name
                     temp_file.seek(0)
-                    transcript = self.local_file_to_transcription(open(temp_file_name, 'rb'))
+
+                    song = AudioSegment.from_mp3(temp_file.name)
+                    ten_minutes = 10 * 60 * 1000
```

Comment:
@jonmc12 should we add `brew install ffmpeg` to the setup flow? the executables for `ffprobe`, which are needed to call `from_mp3` here, are not found when it is downloaded with `pip` alone